src/main/java/ru/skypro/courseWork/dto/CommentDto.java

```diff
@@ -0,0 +1,19 @@
+package ru.skypro.courseWork.dto;
+
+import lombok.Data;
+
+@Data
+public class CommentDto {
+
+    private Integer author;
+
+    private String authorImage;
+
+    private String authorFirstName;
+
+    private Integer createAt;
```

Comment:
```suggestion
    private Long createAt;
```
см. внимательнее спецификацию